src/components/imageOptionsEditor/imageOptionsEditor.js

```diff
@@ -78,35 +89,32 @@ define(['globalize', 'dom', 'dialogHelper', 'emby-checkbox', 'emby-select', 'emb
         });
     }
 
-    function editor() {
-        this.show = function (itemType, options, availableOptions) {
-            return new Promise(function (resolve, reject) {
-                var xhr = new XMLHttpRequest();
-                xhr.open('GET', 'components/imageOptionsEditor/imageOptionsEditor.template.html', true);
-
-                xhr.onload = function (e) {
-                    var template = this.response;
-                    var dlg = dialogHelper.createDialog({
+export class editor {
+    constructor() {
+        this.show = (itemType, options, availableOptions) => {
+            return new Promise((resolve) => {
+                //TODO: remove require
+                require(['text!./components/imageOptionsEditor/imageOptionsEditor.template.html'], template => {
```

Comment:
Does import not work for HTML templates? What's the best way to handle these in the future?

Reply:
`import('text!./components/imageOptionsEditor/imageOptionsEditor.template.html').then(({default: template}) => {`
 
**but i'm not quite sure**

Reply:
No worries, I was just curious.

Reply:
> Does import not work for HTML templates? What's the best way to handle these in the future?

Once we are one with removing RequireJS, this will be handled by webpack loaders. Just import the HTML file like an ES6 module and Webpack will take care of it.